OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -128,4 +136,21 @@ public WRot LocalOrientation(Actor self)
 			return new WRot(WAngle.Zero, WAngle.Zero, WAngle.FromFacing(facing));
 		}
 	}
+
+	public class TurretFacingInit : IActorInit<int>
+	{
+		[FieldFromYamlKey] readonly int value = 128;
+		public TurretFacingInit() { }
+		public TurretFacingInit(int init) { value = init; }
+		public int Value(World world) { return value; }
+	}
+
+	public class TurretFacingsInit : IActorInit<Dictionary<string, int>>
```

Comment:
Most units only have one turret, so it would be a bit ugly to require them to use the more verbose TurretFacingsInit.